sympy/physics/control/lti.py

```diff
@@ -355,10 +355,8 @@ def is_stable(self):
         >>> tf3 = TransferFunction(4, q*s - r, s)
         >>> tf3.is_stable()
         False
-
-        # not enough info about the symbols to determine stability
```

Comment:
I think
```
>>> # not enough info about the symbols to determine stability
```
would also work here, as would making this not be a comment at all and adding a trailing newline
```
Note this returns None if there is not enough info about the symbols to determine stability

```